searchcorespi/CMakeLists.txt

```diff
@@ -18,8 +18,4 @@ vespa_define_module(
     src/vespa/searchcorespi
     src/vespa/searchcorespi/flush
     src/vespa/searchcorespi/index
-
```

Comment:
Consider removing this file.

Reply:
Yes, forgot to remove searchcorespi directory